dist/formio-complete.js

```diff
@@ -63725,7 +63732,7 @@ module.exports = function(app) {
           format: 'yyyy-MM-dd HH:mm a',
           enableDate: true,
           enableTime: true,
-          defaultDate: '',
+          defaultDate: 'moment.now()',
```

Comment:
Is there a reason we changed the default settings to now? This is a change of functionality so people will probably notice it.